cmd/server/flags.go

```diff
@@ -271,6 +271,12 @@ var flags = []cli.Flag{
 		Name:    "github-skip-verify",
 		Usage:   "github skip ssl verification",
 	},
+	&cli.StringFlag{
+		EnvVars: []string{"WOODPECKER_GITHUB_RELEASE_ACTIONS"},
+		Name:    "github-release-actions",
+		Usage:   "On which actions to trigger a release pipeline",
+		Value:   "prereleased, released",
+	},
```

Comment:
Instead of limiting here I would suggest to inject an environment variable to each step `CI_RELEASE_TYPE=draft,pre-release,release`

Reply:
I would guess this could be pretty simple by introducing #312